Gini-iOS-SDK/GINIHTTPError.h

```diff
@@ -15,20 +19,28 @@
  */
 @interface GINIHTTPError : NSError
 
-+ (instancetype)HTTPErrrorWithResponse:(GINIURLResponse *)response;
+/** @name Factories */
 
+/**
+* Factory to create a new GINIHTTPError instance.
+*
+* @param code      The error code for the error.
+* @param userInfo  The userInfo for the error.
+*/
++ (instancetype)HTTPErrrorWithCode:(NSInteger)code userInfo:(NSDictionary *)userInfo;
```

Comment:
Shiver me timbers, HTTPErrrarrrrr!

Convenience methods fangen in der regel mit kleinen Buchstaben an. HTTP können wir weglassen:
```
[GINIHTTPError errorWithCode: userInfo:]
```